storybook/src/components/advanced-select/advanced-select.core-template.ts

```diff
@@ -0,0 +1,22 @@
+import { AdvancedSelect } from "dso-toolkit";
+import { html } from "lit-html";
+
+import { ComponentImplementation } from "../../templates";
+import { ifDefined } from "lit-html/directives/if-defined.js";
+
+export const coreAdvancedSelect: ComponentImplementation<AdvancedSelect<unknown>> = {
+  component: "advancedSelect",
+  implementation: "core",
+  template: () =>
+    function advancedSelectTemplate({ options, active, activeHint, open, dsoClick, dsoOptionClick, dsoRedirectClick }) {
+      return html`<dso-advanced-select
+        .options=${options}
+        .active=${ifDefined(active)}
+        .activeHint=${ifDefined(activeHint)}
+        .open=${open}
```

Comment:
FYI:

Stencil maakt `@Prop()`s van het type `boolean` een boolean attribute. Dat betekent dat de aanwezigheid en afwezigheid van dit attribuut als true en false worden gezien.

Stencil synchroniseert properties en attributen. Je kan deze prop dus op twee manieren zetten: Als property met `.open=${true}` of als boolean attribute als `?open=${true}`.

Hoewel beide werkt heb ik voorkeur voor `?open=${true}` omdat het een simpel type is. Er wordt geen complex object doorgegeven, dat kan namelijk niet als attribuut.